api/Sanitizer.json

```diff
@@ -0,0 +1,196 @@
+{
+  "api": {
+    "Sanitizer": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Sanitizer",
+        "spec_url": "https://wicg.github.io/sanitizer-api/#sanitizer-api",
+        "support": {
+          "chrome": {
+            "version_added": "105",
```

Comment:
We should start from clean - the reason we deleted this is we don't want this abortive history.

Reply:
We discussed it at BCD meeting and agreed that keeping the history of Chrome supporting the previous version of the spec for a while is still useful. It might be a completely new PR, but some info from this one should be used.

Reply:
Thanks @pepelsbey That's a reversion of their previous position, and seems "nuts". 

@Elchi3 This makes no sense - no one should be using the older version and we previously agreed it was dead and gone. That makes sense because no one should be using it, and trying to build on top of that will make this a very ugly compat story.